etc/init.d/lncm-post

```diff
@@ -48,10 +48,53 @@ start() {
                 if [ $ONLINE -eq 0 ]; then
                     echo "Online, proceeding with post-installation"
 
-                    echo "Install docker-compose"
+                    echo "Install docker-compose and dependencies for config files"
+                    # Install dependencies for docker compose and for generating config files
                     /sbin/apk add py3-pip
+                    /sbin/apk add curl
+                    /sbin/apk add pwgen
+
                     /usr/bin/pip3 install --upgrade pip
                     /usr/bin/pip3 install docker-compose
+                    
+                    cd /home/lncm
+                    
+                    # Generate RPC USER and PASS
+                    GENERATEDUID=`pwgen -s 12 1` # This can be used in for for subsequent scripts
```

Comment:
@meeDamian Has a script that doesn't require a dependency to generate pw from word lists

Reply:
I think PWGEN will suffice for now. We can change it later?